### PR TITLE
Fixed case where partition is created,  but mount points not added

### PR DESCRIPTION
**Pull request by kdvlr**

### CODE REVIEW

File: setup/pi/create-backingfiles-partition.sh [hunk 1]
```diff
@@ -38,24 +37,24 @@ then
...
+  else
+    echo "LABEL=backingfiles $BACKINGFILES_MOUNTPOINT xfs auto,rw,noatime 0 2" >> /etc/fstab
+  fi
+  if grep -q 'mutable' /etc/fstab
+  then
+    setup_progress "mutable already defined in /etc/fstab. Not modifying /etc/fstab."
+  else
+    echo "LABEL=mutable $MUTABLE_MOUNTPOINT ext4 auto,rw 0 2" >> /etc/fstab
   fi
+  setup_progress "Mount Points updated /etc/fstab."
```
Comment (marcone) [inline setup/pi/create-backingfiles-partition.sh]: it will log this every time, even when it didn't update /etc/fstab

Comment (kdvlr) [reply setup/pi/create-backingfiles-partition.sh]: What would be a better log message here? ```Partitions setup complete.```  ?

. I want there to be a clear statement that says create-backingfiles-partition  is over.

Comment (marcone) [reply setup/pi/create-backingfiles-partition.sh]: I'm a fan of "Done" myself :)

Comment (kdvlr) [reply setup/pi/create-backingfiles-partition.sh]: Oh well.. Brevity is not my strong point. Modified it.

**Pull request merged**